.golangci.yml

```diff
@@ -11,5 +14,39 @@ linters:
     - gosimple
     - govet
     - misspell
+    - revive
     - staticcheck
     - unused
+
+linters-settings:
+  revive:
+    rules:
+      - name: blank-imports
+      - name: context-as-argument
+      - name: context-keys-type
+      - name: dot-imports
+        arguments:
+          - allowedPackages:
+            - "github.com/onsi/ginkgo"
+            - "github.com/onsi/ginkgo/v2"
+            - "github.com/onsi/gomega"
+      - name: empty-block
+      - name: error-naming
+      - name: error-return
+      - name: error-strings
+      - name: errorf
```

Comment:
for some reason, if rule is not declared, its disabled by default. I am not sure if there is a to just define disabled rules here